modules/rtm/tests/testthat/test.invert_simple.R

```diff
@@ -1,6 +1,16 @@
 if (interactive()) {
   devtools::load_all("~/Projects/pecan/pecan/modules/rtm")
+} else {
+    library(PEcAnRTM)
 }
+
+
+# Skip computationally intensive test on TRAVIS
+travis <- Sys.getenv('TRAVIS') == 'true'
+if (travis) {
+    message('Skip computationally intensive inversion on Travis')
+} else { 
+
```

Comment:
`skip()` just throws an error (effectively the same thing as `stop()`) when run outside of a `test_that(...)` block. The way these tests are set up, they first run simulations and then `test_that(...)` on the outputs.